docs/migration-recipe.md

```diff
@@ -0,0 +1,106 @@
+Migrating to new host:
+----------------------
+
+### Prepare for migration:  
+
+1.  Ensure that salt-master, loadbalancer, and host in question are up, and that their health check for the relevant service is failing in haproxy after the host is fully up
+
+To view haproxy status: 
+
+-vagrant up the salt-master, loadbalancer, and host in question (`vagrant up`)
+
+-`vagrant ssh-config salt-master loadbalancer >> vagrant-ssh Ssh -L 4646:127:0.0.1:464 -F vagrant-ssh loadbalancer`
```

Comment:
Describe what this is doing. I also think there are two commands here, let's break those up!

Something like:

```
Prepare an ssh configuration file for accessing the hosts with native ssh commands rather than vagrant ssh:
`vagrant ssh-config salt-master loadbalancer >> vagrant-ssh`

Open an ssh session with port forwarding to the haproxy status page:
`Ssh -L 4646:127:0.0.1:464 -F vagrant-ssh loadbalancer`
```